src/plugins/discover/public/application/components/chart/hits_counter/hits_counter.tsx

```diff
@@ -61,14 +69,20 @@ export function HitsCounter({ hits, showResetButton, onResetQuery }: HitsCounter
       >
         <EuiFlexItem grow={false}>
           <EuiText size="s">
-            <strong data-test-subj="discoverQueryHits">{hits.toLocaleString()}</strong>{' '}
-            <FormattedMessage
-              id="discover.hitsPluralTitle"
-              defaultMessage="{hits, plural, one {hit} other {hits}}"
-              values={{
-                hits,
-              }}
-            />
+            <strong>
+              <FormattedMessage
```

Comment:
nit: Do we want to add a aria-label for the entire results display to help screen readers understand the relationship between these numbers?

Reply:
yeah that's a good idea, i'll add